modules/mayaUSD.mod.template

```diff
@@ -8,7 +8,7 @@ PYTHONPATH+:=lib/python
 USD_LOCATION:=
 
 + MayaUSD_LIB ${MAYAUSD_VERSION} ${CMAKE_INSTALL_PREFIX}
-icons: 
+icons: lib/icons
```

Comment:
All the outliner icons above were added to this folder (as part of the base lib). So that any of the 3 plugins will get them.